.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.codeActionsOnSave": {
-    "source.formatDocument": true,
-    "source.fixAll.eslint": true
+    "source.formatDocument": "explicit",
```

Comment:
Can you add the entire `.vscode` folder into the gitignore. I think I might have accidentally added this at some point.